packages/headless/src/controllers/commerce/search/facets/headless-search-date-facet.ts

```diff
@@ -0,0 +1,32 @@
+import {CommerceEngine} from '../../../../app/commerce-engine/commerce-engine';
+import {executeSearch} from '../../../../features/commerce/search/search-actions';
+import {loadReducerError} from '../../../../utils/errors';
+import {
+  CommerceDateFacet,
+  buildCommerceDateFacet,
+} from '../../facets/core/date/headless-commerce-date-facet';
+import {CoreCommerceFacetOptions} from '../../facets/core/headless-core-commerce-facet';
+import {loadSearchReducer} from '../utils/load-search-reducers';
+
+export type SearchDateFacetOptions = Omit<
```

Comment:
This is my own doing, but we're using that exact same interface (under different names) for all of these controllers (product listing / search regular, date, numeric, and soon category facets, i.e., 8 times). I think it would be worth defining it once somewhere and reuse it 😄